nav_msgs/Path[] plan_1
nav_msgs/Path[] plan_2
nav_msgs/Path[] plan_3